docs/service-catalog/docs/050-gs-broker-registration.md

```diff
@@ -0,0 +1,67 @@
+---
+title: How to register a broker
+type: Getting Started
+---
+
+This Getting Started guide shows how to register a new broker in the Service Catalog. The broker can be either a Namespace-scoped ServiceBroker or a cluster-wide ClusterServiceBroker. Follow this guide to register a cluster-wide [UPS Broker](https://github.com/kubernetes-incubator/service-catalog/tree/master/charts/ups-broker) in the Service Catalog.
+
+## Prerequisites
+
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl)
+* [helm](https://github.com/helm/helm#install)
+
+## Steps
+
+1. Clone the [`service-catalog`](https://github.com/kubernetes-incubator/service-catalog) repository to your local machine:
+    ```
+    git clone https://github.com/kubernetes-incubator/service-catalog.git
+    ```
+
+2.  Run this command to install the chart with the `ups-broker` name in the `ups-broker` Namespace:
```

Comment:
with the `ups-broker` name - do you give that chart this name? 
If the chart exists with that name already, I think you should say "Install the {NAME} chart". Otherwise it sounds like you set its name when you execute this command.

Reply:
Yup, you can give this chart a name and name it as you wish. Confirmed with the devs.